hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieCompactor.java

```diff
@@ -50,13 +42,14 @@
    * @throws IOException when encountering errors
    */
   HoodieCompactionWorkload generateCompactionWorkload(JavaSparkContext jsc,
```

Comment:
Isn't the fileIds something that can be computed inside this method ? Why do we need to pass this into the method ?

Reply:
I had removed the CompactionId from this method. Not sure if you were referring to any other field